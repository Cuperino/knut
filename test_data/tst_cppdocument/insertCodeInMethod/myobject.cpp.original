#include "myobject.h"

#include <iostream>


MyObject::MyObject(const std::string& message)
    : m_message(message)
{}

MyObject::~MyObject()
{}

void MyObject::sayMessage() {
    std::cout << m_message << std::endl;
}

static void displayString(const std::string& test) {
    std::cout << test << std::endl;
}

void MyObject::sayMessage(const std::string& test) {
    displayString(test);
}
