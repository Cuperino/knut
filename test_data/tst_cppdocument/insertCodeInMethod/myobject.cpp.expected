#include "myobject.h"

#include <iostream>


MyObject::MyObject(const std::string& message)
    : m_message(message)
{}

MyObject::~MyObject()
{}

void MyObject::sayMessage() {
    // added this new line at the start of this method
    std::cout << m_message << std::endl;
    // added this new line at the end of this method
}

static void displayString(const std::string& test) {
    // added this new line at the start of this method
    std::cout << test << std::endl;
    // added this new line at the end of this method
}

void MyObject::sayMessage(const std::string& test) {
    displayString(test);
}
